bip32/Cargo.toml

```diff
@@ -29,7 +29,7 @@ zeroize = { version = "1", default-features = false }
 k256 = { version = "0.13", optional = true, default-features = false, features = ["ecdsa", "sha256"] }
 once_cell = { version = "1", optional = true }
 pbkdf2 = { version = "0.12", optional = true, default-features = false, features = ["hmac"] }
-secp256k1-ffi = { package = "secp256k1", version = "0.27", optional = true }
+secp256k1-ffi = { package = "secp256k1", version = "0.29", optional = true, default-features = false, features = ["alloc"] }
```

Comment:
The version bump is a breaking change. It would be good to separate this out into two separate changes:
- A PR that disables the `std` feature (backwards-compatible, necessary to fix `bip32`'s no-std compatibility, can be published in a point release).
- A subsequent PR that bumps the `secp256k1` version (breaking change, needs coordination).